### DISCUSSION TITLE
Cupertino design for iOS and MacOs

### DISCUSSION THREAD

**Discussion by AlleSchonWeg** [AvaloniaUI/Avalonia#14912] [Ideas]
### Is your feature request related to a problem? Please describe.

Hi,
it would be nice for iOS and macOS to draw the UI in an native look and feel. In my opinion this is what Apple users want.

### Describe the solution you'd like

Cupertino design should be available like metal or fluent.

**Comment by MikeCodesDotNET** [comment]
I started on this long ago, but it's a huge amount of work.

It's something best left to the community to create rather than the core team. 

![image](https://github.com/AvaloniaUI/Avalonia/assets/552074/3f386643-4ee4-4309-b4e9-28897d3faf76)

**Comment by AlleSchonWeg** [comment]
Perhaps Avalonia can convert/copy the themes from flutter. I don't know if this is possible.

**Comment by timunie** [comment]
It's possible but nothing we plan to do. Check awsome-avalonia for how other style libs look like. If someone wants to create such a theme, it can be added to awsome-avalonia.

**Comment by thevortexcloud** [comment]
You can make the controls look like pretty much anything. Someone made a Windows 7 theme awhile back.

https://github.com/Splitwirez/avalonia-aero-theme

There is also this theme that looks kind of like iOS: https://github.com/kikipoulet/CherylUI

But making things look native is a lot of work. A lot of design languages are not publicly documented.

**Comment by dhervieux** [comment]
You can follow the progress of the macOS theme here: 
https://github.com/Devolutions/avalonia-themes

If you're interested, you can also contribute.
